### DISCUSSION TITLE
Viseron Writing as root

### DISCUSSION THREAD

**Discussion by ItsMeRitch** [roflcoopter/viseron#498] [Q&A] [answered]
Hi all,

Very new to the whole NVR game. 

Is there a way to get Viseron to stop writing as root in the attached volumes/binds? 

Thanks

**Comment by roflcoopter** [comment, marked as answer]
Sorry totally missed this one!

Yes, you can set the user and group ID to be used in the container: https://viseron.netlify.app/docs/documentation/installation#user-and-group-identifiers

I run with `PUID=1000` and `PGID=1000` which matches my host user
